src/components/pages/_document.js

```diff
@@ -27,6 +27,39 @@ class Document extends NextDocument {
       sheet.seal();
     }
   }
+
+  render() {
+    return (
+      <Html>
+        <Head>
+          <link
+            key="icon"
+            rel="icon"
+            type="image/png"
+            sizes="32x32"
+            href="/favicon.png"
+          />
+          {/* eslint-disable-next-line @next/next/no-page-custom-font */}
+          <link
+            key="fonts"
+            href="https://fonts.googleapis.com/css?family=Open+Sans:400italic,600italic,700italic,300,400,700&display=optional"
```

Comment:
Do you need all these different italic styles?

Reply:
Good question, I'll take a look. These were used in the old app that's why we import them, but I think for now we don't use them